recipes/contributing.md

```diff
@@ -1,12 +1,11 @@
-## Contributions
+# Contributions
```

Comment:
@chicco785 I like the idea of keeping only 1 level-1 title per file to give content a tree-like structure.

By the way, I found [this gem](https://github.com/google/styleguide/tree/gh-pages/docguide) some weeks ago while updating docs for QL. I think it's worth sharing now that we're going through these kind of fixes in all the repos.